File: DoneItAPI/DoneItAPI/Models/RegistroRequest.cs
namespace DoneItAPI.Models
{
    public class RegistroRequest
    {
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Email { get; set; }
        public string Fecha_Nacimiento { get; set; } // formato "yyyy-MM-dd"
        public string Nombre_Usuario { get; set; }
        public string Password { get; set; }
    }
}

File: DoneIt_Version_Movil/app/src/main/java/com/herreroco/doneit_version_movil/model/RegistroRequest.kt
package com.herreroco.doneit_version_movil.model

data class RegistroRequest(
    val nombre: String,
    val apellido: String,
    val email: String,
    val fecha_nacimiento: String, // Formato YYYY-MM-DD
    val nombre_usuario: String,
    val password: String
)
